app/views/merchants/registrations/new.html.erb

```diff
@@ -1,30 +1,50 @@
-<div class="container">
-  <h2>Sign up </h2>
-  <br/><br/>
-  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-      <%= devise_error_messages! %>
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
Why are you adding Bootstrap CDN file when it is already loaded in application.css?